### ISSUE TITLE
Decryption of json-backup

### ISSUE THREAD

**Issue by jonasryberg** [keshav-space/safenotes#216]
Hi!

Is it possible to decrypt the posts in the JSON backup file using another tool, such as OpenSSL? If so, what information would I need?

**Comment by IAAJL** [comment]
Yes. I am also curious. Normally the encryption is using a passphrase. But here no passphrase is being requested. So i am not fully sure of its security
